### DISCUSSION TITLE
How to run 'next export' without minify or uglify

### DISCUSSION THREAD

**Discussion by panacholn** [vercel/next.js#11911] [Help]
I want to export the project with 'next export' command without any minify or uglify the code

**Comment by timneutkens** [comment]
Why do you want to do that? It'll cause significantly larger bundles to be shipped to users.

**Reply by mutalis** [reply]
I'm also interested to export the project without any minify or uglify the code in order to be able to deliver it to any person without technical JS expertise. So please any help on this matter is appreciated. @panacholn @timneutkens
